### PR TITLE
add mail and mail sending task config

### PR DESCRIPTION
**Pull request by mdayican**
https://tools.hmcts.net/jira/browse/BPS-1369

### Change description ###

add mail and mail sending task config.
using existing report recipients and mail-from.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -146,6 +149,9 @@ proxy:
 
 reconciliation:
   api-key: ${RECONCILIATION_API_KEY}
+  report:
+      mail-from: ${SMTP_USERNAME}
+      mail-recipients: ${REPORTS_RECIPIENTS}
```
Comment (mdayican) [inline src/main/resources/application.yaml:154]: using same values like other reports

Comment (doncem) [reply src/main/resources/application.yaml:154]: i guess we want to keep it separate in case recipients need to be different

**Review by doncem — APPROVED** [review]

File: src/integrationTest/resources/application.properties [hunk 2]
```diff
@@ -47,6 +47,9 @@ scheduling.task.create-reconciliation-summary-report.cron=0/10 * * * * *
 scheduling.task.create-reconciliation-detailed-report.enabled=false
 scheduling.task.create-reconciliation-detailed-report.cron=0/10 * * * * *
 
+scheduling.task.send-reconciliation-report-mail.enabled=false
+scheduling.task.send-reconciliation-report-mail.cron=0/10 * * * * *
```
Comment (rkondratowicz) [inline src/integrationTest/resources/application.properties:51]: every 10 seconds?

**Review by rkondratowicz — APPROVED** [review]

File: src/main/resources/application.yaml (hunk 1 above)
Comment (mdayican) [reply src/main/resources/application.yaml:154]: that is why I created different properties for now.

**Pull request merged**